App/Redux/PhotoViewingRedux.ts

```diff
@@ -56,53 +57,66 @@ const actions = {
 export type PhotoViewingAction = ActionType<typeof actions>
 
 export interface ThreadData {
-  readonly thread: Thread
+  readonly id: ThreadId
+  readonly name: string
   readonly querying: boolean
   readonly photos: ReadonlyArray<Photo>
   readonly error?: Error
 }
 
 interface ThreadMap {
-  readonly [key: string]: ThreadData
+  readonly [key: string]: ThreadData | undefined
 }
 
 interface PhotoViewingState {
-  readonly threads: ThreadMap,
+  readonly navigateToNewThread: boolean
+  readonly shareToNewThread?: {
+    threadName: string
+    imageId: PhotoId
+    comment?: string
+  }
```

Comment:
Above is where we keep track of requests to navigate to a new thread once it is created and share a wallet photo to a new thread once it was created.